tests/runtime/conftest.py

```diff
@@ -235,6 +237,29 @@ def other_task_model_id(other_good_model_path) -> str:
     model_manager.unload_model(model_id)
 
 
+def register_trained_model(
+    servicer: Union[RuntimeGRPCServer, GlobalPredictServicer],
+    model_id: str,
+    training_id: str,
+):
+    """Helper to auto-load a model that has completed training. This replaces
+    the old auto-load feature which was only needed for unit tests
+    """
+    model_future = MODEL_MANAGER.get_model_future(training_id)
+    model = model_future.load()
+    loaded_model = (
+        LoadedModel.Builder()
+        .id(model_id)
+        .type("trained")
```

Comment:
oh this is yet another gross corner where `model_type` is still just some user defined string :(

This information _should_ be all contained in the `model` here, but this is also only relevant for model-mesh so probably not worth refactoring it out right now

Reply:
Hm, good point. That's a good one to clean up down the road